test-case-reporting/index.ts

```diff
@@ -16,23 +16,72 @@
 
 require('dotenv').config();
 
-import {KarmaReporter} from 'test-case-reporting';
+import {KarmaReporter, TestRun} from 'test-case-reporting';
+import {TestResultRecord} from './src/test_result_record';
+import {dbConnect} from './src/db';
 import express from 'express';
 const app = express();
 const PORT = process.env.PORT || 8080;
+const MAX_PAGE_SIZE = 500;
+
+function renderTestRunList(testRuns: Array<TestRun>): string {
+  return `Rendering ${testRuns.length} test runs!`;
+}
 
 app.get('/', (req, res) => {
   res.send('Hello, world, 2.0!');
 });
 
-app.get('/test-results/pr/:prNumber', (req, res) => {
-  const {prNumber} = req.params;
-  res.send(`List of test cases for PR number ${prNumber}`);
+app.get('/test-results/build/:buildNumber', async (req, res) => {
+  const {buildNumber} = req.params;
+  const {json} = req.query;
+  const db = dbConnect();
+  const testResultRecord = new TestResultRecord(db);
```

Comment:
Let's create the DB connection and the test result record once outside of the request handlers.

Reply:
Oh, I thought I'd already done this.

Done.